styles/less/global/inventory-item.less

```diff
@@ -12,87 +12,151 @@
+                    opacity: 0;
+                }
             }
 
-            .item-tags,
-            .item-labels {
-                display: flex;
-                gap: 10px;
-
-                .tag {
-                    display: flex;
-                    flex-direction: row;
-                    justify-content: center;
-                    align-items: center;
-                    padding: 3px 5px;
-                    font-size: 12px;
+            .item-label {
+                flex: 1;
+                font-family: @font-body;
+                align-self: center;
 
-                    background: light-dark(@dark-15, @beige-15);
-                    border: 1px solid light-dark(@dark, @beige);
-                    border-radius: 3px;
+                .item-name {
+                    font-size: 14px;
                 }
 
-                .label {
+                .item-tags,
+                .item-labels {
                     display: flex;
-                    flex-direction: row;
-                    justify-content: center;
-                    align-items: center;
-                    font-size: 12px;
-                    gap: 4px;
+                    gap: 10px;
+
+                    .tag,
+                    .label {
+                        display: flex;
+                        flex-direction: row;
+                        justify-content: center;
+                        align-items: center;
+                        font-size: 12px;
+                    }
+
+                    .tag {
+                        padding: 3px 5px;
+                        background: light-dark(@dark-15, @beige-15);
+                        border: 1px solid light-dark(@dark, @beige);
+                        border-radius: 3px;
+                    }
+
+                    .label {
+                        gap: 4px;
+                    }
                 }
             }
+
+            .item-resource {
+                flex: 0 0 60px;
+                display: flex;
+                align-items: center;
+                gap: 4px;
+            }
+            .controls {
+                flex: 0 0 auto;
+                display: flex;
+                align-items: center;
+                justify-content: end;
+                gap: 8px;
+            }
         }
 
-        .controls {
-            display: flex;
-            align-items: center;
-            justify-content: end;
-            gap: 8px;
+        .inventory-item-content {
+            &.extensible {
+                max-height: 0;
+                overflow: hidden;
+                transition: max-height 2s linear;
+                &.extended {
+                    max-height: 1000px; // Reasonable large value
+                    //maybe need the redo this.... idk, i'm just a little wittle girl, atte: JP.
+                }
+            }
+            .item-resources {
+                display: flex;
+                align-items: center;
+                gap: 4px;
```

Comment:
Think we need to add maybe `margin-top: 4px;` for the resource container.
Spiky, evil d4s are kinda of scraping the top.
<img width="680" height="135" alt="image" src="https://github.com/user-attachments/assets/09fdfd77-f8e3-4133-bf58-9a781678bb53" />